packages/render/src/use-frame.tsx

```diff
@@ -325,6 +331,7 @@ export function useFrame<
       homeframeUrl,
       signerState.hasSigner,
       signerState.signer,
+      onErrorRef,
```

Comment:
got a eslint warning here about missing dep, since it is ref it should be fine to be included.